ERCS/erc-6900.md

```diff
@@ -462,17 +463,10 @@ interface IExecutionHookModule is IModule {
 }
 ```
 
-### Validation Functions and Their Installation/Uninstallation
-
```

Comment:
Do we need to include the following points under the validation installation section?
- An account can have multiple validation modules/functions installed.
- An account can have the same validation module installed multiple times.

Reply:
I'm not quite following why those aspects were removed. I think this one is also worth keeping:
* The entity ID of a validation function installed on an account MUST be unique.

Reply:
> An account can have multiple validation modules/functions installed.
An account can have the same validation module installed multiple times.

These are implicitly true from the spec's interfaces - I thought we didn't need to call them out.

> The entity ID of a validation function installed on an account MUST be unique.

I'm not sure how this got into the spec, since this isn't the case for the RI right now. We should remove this.